osquery/filesystem/darwin/bsd_file_flags.cpp

```diff
@@ -19,15 +19,24 @@
 namespace osquery {
 namespace {
 /// The list of supported flags, as documented in `man 2 chflags`
+/// And in https://github.com/apple/darwin-xnu/blob/master/bsd/sys/stat.h
 const std::map<std::uint32_t, std::string> kBsdFlagMap = {
-    {UF_NODUMP, "NODUMP"},
-    {UF_IMMUTABLE, "UF_IMMUTABLE"},
-    {UF_APPEND, "UF_APPEND"},
-    {UF_OPAQUE, "OPAQUE"},
-    {UF_HIDDEN, "HIDDEN"},
-    {SF_ARCHIVED, "ARCHIVED"},
-    {SF_IMMUTABLE, "SF_IMMUTABLE"},
-    {SF_APPEND, "SF_APPEND"}};
+    {UF_APPEND, "UF_APPEND"}, // 0x00000004
+    {UF_COMPRESSED, "COMPRESSED"}, // 0x00000020
+    {UF_DATAVAULT, "DATAVAULT"}, // 0x00000080
+    {UF_HIDDEN, "HIDDEN"}, // 0x00008000
+    {UF_IMMUTABLE, "UF_IMMUTABLE"}, // 0x00000002
+    {UF_NODUMP, "NODUMP"}, // 0x00000001
+    {UF_OPAQUE, "OPAQUE"}, // 0x00000008
+    {UF_TRACKED, "TRACKED"}, // 0x00000040
+
+    {SF_APPEND, "SF_APPEND"}, // 0x00040000
+    {SF_ARCHIVED, "ARCHIVED"}, // 0x00010000
```

Comment:
`SF_ARCHIVED` perhaps for consistency? It seems most super-user flags are prefixed with `SF`

Reply:
That one is pre-existing, and I didn't want to change the format